src/components/card/card.js

```diff
@@ -128,3 +129,41 @@ function mdCardDirective($mdTheming) {
     }
   };
 }
+
+
+/**
+ * @ngdoc directive
+ * @name mdPlainImage
+ * @module material.components.card
+ *
+ * @restrict A
+ *
+ * @description
+ * By default `<md-card>` stretches all images in order for them fit them nicely inside
+ * the card. This can cause low-resolution images to appear blurry. You can add the
+ * `md-plain-image` attribute to a `<img>` tag if you want to avoid the stretching for that
+ * particular image.
+ *
+ * @usage
+ * ### Card with a plain image
+ * <hljs lang="html">
+ * <md-card>
+ *  <img src="card-image.png" alt="image caption" md-plain-image>
+ * </md-card>
+ * </hljs>
+ */
+function mdPlainImageDirective($mdUtil) {
+  return {
+    restrict: 'A',
+    link: function ($scope, $element) {
```

Comment:
Just tried it out, it throws the browser into an infinite loop if it gets used within a `ngRepeat`.